### DISCUSSION TITLE
Sobre o andamento da documentação.

### DISCUSSION THREAD

**Discussion by ivannaprata** [EasyRoomCompany/Guardian#66] [Ideas]
@mikaelmonteirodev como está o andamento da documentação? Até agora tenho achado muito bom o que já está feito.

**Comment by mikaelmonteirodev** [comment]
Estou providenciando três documentações, para ficar mais organizado. São  elas:
1. Documentação geral da aplicação para uso;
2. Documentação técnica para Frontend; e
3. Documentação técnica para Backend.

**Comment by kevinDsousa** [comment]
@mikaelmonteirodev estou finalizando as rotas de backend, quando concluir te passo o que é possivel documentar
